ui/packages/catalog-ui-search/src/main/webapp/react-component/container/map-settings/map-settings.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+import MapSettingsPresentation from '../../presentation/map-settings'
+import Dropdown from '../../presentation/dropdown'
+import { hot } from 'react-hot-loader'
+const user = require('../../../component/singletons/user-instance.js')
+const mtgeo = require('mt-geo')
+
+const exampleLat = '14.94',
```

Comment:
Can we move this example data into `ExampleCoordinates.tsx`?

Reply:
I think that would be better